v2/workloadapi/client.go

```diff
@@ -353,6 +347,49 @@ func defaultClientConfig() clientConfig {
 	}
 }
 
+func parseX509Context(resp *workload.X509SVIDResponse) (*X509Context, error) {
+	svids, err := parseX509SVIDs(resp, -1)
+	if err != nil {
+		return nil, err
+	}
+
+	bundles, err := parseX509Bundles(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &X509Context{
+		SVIDs:   svids,
+		Bundles: bundles,
+	}, nil
+}
+
+// parseX509SVIDs parses between 1 and n SVIDs from the response. It fails
+// if it cannot parse at least one, since the SVID is required in the response
+// and the spec indicates clients SHOULD reject such messages. If n is less
+// than one, then all SVIDs are returned.
+func parseX509SVIDs(resp *workload.X509SVIDResponse, n int) ([]*x509svid.SVID, error) {
```

Comment:
looks like only uses for it is -1 or 1, maybe we can use bool here?

Reply:
That's a fine suggestion. I'll change it.